### PR TITLE
[READY] - Update action due to node 12 deprecation

### LINKED ISSUE #5: Octkit seems to be causing deprecation warning
## Description

v1.0 of this module

Seems to be with the `@octokit` vendored module 

```
Run nebulaworks/action-standardize-pr-comment@v1.0
[@octokit/rest] `const Octokit = require("@octokit/rest")` is deprecated. Use `const { Octokit } = require("@octokit/rest")` instead
```
Source: https://github.com/Nebulaworks/action-standardize-pr-comment/actions/runs/4857963669/jobs/8658915870

Originally noticed here: https://github.com/socallinuxexpo/scale-network/actions/runs/4857912351/jobs/8658822126

### PR DESCRIPTION
**Pull request by sarcasticadmin**
## Description

Fixes: https://github.com/Nebulaworks/action-standardize-pr-comment/issues/5

Upgrading to latest support node version (node 16) for GHA

## Previous Behavior
- node 12
- mixed results for existing package scripts

## New Behavior
- package script: `check-all` is working
- fmt `src/`
- init nix.flake to provide node 16
- update vendored `node_modules` :cry: 
- bump version to v2.0.0

## Tests

`self test` is working: https://github.com/Nebulaworks/action-standardize-pr-comment/actions/runs/5203945793

`check-all` is passing:
```
$ yarn check-all
yarn run v1.22.19
$ concurrently "npm:format-check" "npm:lint" "npm:test" "npm:build"
[lint] 
[lint] > standardize-pr-comment@2.0.0 lint
[lint] > eslint src/*.js tests/*.js
[lint] 
[format-check] 
[format-check] > standardize-pr-comment@2.0.0 format-check
[format-check] > prettier --check src/*.js tests/*.js
[format-check] 
[test] 
[test] > standardize-pr-comment@2.0.0 test
[test] > jest --testTimeout 10000
[test] 
[build] 
[build] > standardize-pr-comment@2.0.0 build
[build] > node src/main.js; exit 0
[build] 
[build] ::error::standardize-pr-comment can only be run with comment events
[build] npm run build exited with code 0
[format-check] Checking formatting...
[lint] npm run lint exited with code 0
[format-check] All matched files use Prettier code style!
[format-check] npm run format-check exited with code 0
[test]  PASS  tests/main.test.js
[test]   ✓ translate issue comment (2 ms)
[test]   ✓ translate PR review comment
[test] 
[test] Test Suites: 1 passed, 1 total
[test] Tests:       2 passed, 2 total
[test] Snapshots:   0 total
[test] Time:        0.327 s, estimated 1 s
[test] Ran all test suites.
[test] npm run test exited with code 0
Done in 1.60s.
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Standardize PR Comment
 
+## Test
```
Comment (sarcasticadmin) [inline README.md]: self-test should be successful

File: action.yml [hunk 2]
```diff
@@ -12,5 +12,5 @@ outputs:
   url:
     description: 'The URL of the pull request'
 runs:
-  using: 'node12'
-  main: 'src/main.js'
\ No newline at end of file
+  using: 'node18'
```
Comment (sarcasticadmin) [inline action.yml]: self-test should be successful

File: action.yml [hunk 3]
```diff
@@ -12,5 +12,5 @@ outputs:
   url:
     description: 'The URL of the pull request'
 runs:
-  using: 'node12'
-  main: 'src/main.js'
\ No newline at end of file
+  using: 'node16'
```
Comment (sarcasticadmin) [inline action.yml:15]: self-test should be successful

Comment (sarcasticadmin) [inline action.yml:15]: self-test should be successful

File: README.md [hunk 4]
```diff
@@ -1,5 +1,6 @@
 # Standardize PR Comment
```
Comment (sarcasticadmin) [inline README.md]: self-test should be successful

Comment (sarcasticadmin) [reply README.md]: :partying_face: https://github.com/Nebulaworks/action-standardize-pr-comment/actions/runs/5194962979/jobs/9367198868

File: .eslintrc.json [hunk 5]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (sarcasticadmin) [inline .eslintrc.json:1]: self-test should be successful

**Review by davido-nw — APPROVED** [review]
:boom:

**Pull request merged**